api/docs/intro.dox

```diff
@@ -1922,6 +1922,8 @@ DynamoRIO, rebooting in safe mode will allow recovery.
  - The DynamoRIO header files have typedefs that may conflict with other
    header files wrapped in ifndef DR_DO_NOT_DEFINE_<type> to make it
    easier to work around such conflicts.
+ - Alternatively, use DR_DEFINE_FOR_<type>, in which case you may need to
```

Comment:
This doesn't make sense as a separate bullet item: "Alternatively" refers to the prior bullet, not anything else in this list.  Please merge into the prior one.

Reply:
Done